src/main/kotlin/com/terraformation/backend/customer/db/UserStore.kt

```diff
@@ -247,13 +247,17 @@ class UserStore(
         usersDao.fetchOneById(model.userId)
             ?: throw IllegalStateException("Current user not found in users table")
 
+    // If the country code is set, include it in the locale.
+    val newLocale = model.locale?.let { Locale.of(it.language, model.countryCode ?: it.country) }
```

Comment:
just curious what happens when the country doesn't apply to the locale.. like es-ZH for example.., guessing it's a no-op and we default to `es` where applicable?

Reply:
unrelated, do we want to be able to clear the country?

Reply:
If there are no country-specific rules for a language+country pair, it will fall back on the default rules for the language.

And good point on clearing the country. I had that there for testing and forgot to remove it.